core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/devfile/UserDevfile.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.core.model.workspace.devfile;
+
+/** Devfile that persisted in permanent storage. */
+public interface UserDevfile extends Devfile {
```

Comment:
I just realized that this is called `UserDevfile` but it has no reference to a "user". Does it make sense ask questions like "what devfiles belong to this user"? How would we answer those?

Reply:
The reference to the user will appear in this or https://github.com/skabashnyuk/che/pull/12 with https://github.com/skabashnyuk/che/pull/12